src/components/common/BcRegComments.vue

```diff
@@ -0,0 +1,311 @@
+          </div>
+        </section>
+
+        <!-- Confirm Articles Checkbox -->
+        <div class="pr-2">
+          <v-checkbox
+            id="confirm-articles-checkbox"
+            v-model="localConfirmArticles"
+            :label="confirmLabel"
+          />
+        </div>
+      </section>
+    </template>
+
+    <template v-else-if="isBcLimited">
+      <div class="my-6">
+        <p class="subtitle">
+          {{ updatedArticleTitle }}
+        </p>
+        <p class="info-text">
+          Before submitting your alteration notice you must change your company's articles
+          to include a set of BC LTD Articles.
+        </p>
+      </div>
+
+      <section class="text-body-3">
+        <!-- Help Section Toggle -->
+        <HelpSectionToggle
+          :isHelpVisible="helpToggle"
+          @toggle="helpToggle = !helpToggle"
+        />
+
+        <!-- Help Section -->
+        <section
+          v-show="helpToggle"
+          class="mx-8 my-7"
+        >
+          <p class="subtitle mb-2">
+            BC LTD Articles
+          </p>
+          <p class="info-text">
+            Articles should outline the rules and procedures for corporate matters such as holding meetings,
+            issuing and transferring shares, and duties of directors and officers.
+          </p>
+          <div class="provision-help">
+            <span class="info-text">
+              Refer to <span class="font-weight-bold">
+                sections 51.4, 257 and 259 of the Business Corporations Act. </span>
+              Please read these sections
+              carefully before you proceed with filing the Notice of Alteration.
+              Every alteration to the company's Notice of Articles must be appropriately
+              authorized by the company or the court.
+            </span>
+          </div>
+        </section>
+
+        <!-- Confirm Articles Checkbox -->
+        <div class="pr-2">
+          <v-checkbox
+            id="confirm-articles-checkbox"
+            v-model="localConfirmArticles"
+            :label="confirmLabel"
+          />
+        </div>
+      </section>
+    </template>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import HelpSectionToggle from '@/components/common/HelpSectionToggle.vue'
+import { entityTypeInfo } from '@/enums'
+import { CorpTypeCd } from '@bcrs-shared-components/enums'
+import { Component, Prop, Watch } from 'vue-property-decorator'
+
+@Component({
+  components: {
+    HelpSectionToggle
+  }
+})
+export default class BcRegComments extends Vue {
+  @Prop({ default: false }) readonly confirmArticles!: boolean
+  @Prop({ default: false }) readonly isBenefit!: boolean
+  @Prop({ default: false }) readonly isUnlimitedLiability!: boolean
+  @Prop({ default: false }) readonly isCommunityContribution!: boolean
+  @Prop({ default: false }) readonly isBcLimited!: boolean
+  @Prop({ default: '' }) readonly selectedEntityType: CorpTypeCd
+
+  protected helpToggle = false
```

Comment:
Theoretically, protected would allow the template to see this property but it wouldn't be visible outside this class/component.

However that doesn't work, so I'm OK with removing the accessor when the template needs to see this property.